woostarter/patterns/about-row-two-images-offset-text-link.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Title: Two image row, offset text and link
+ * Slug: woostarter/two-image-row-offset-text-link
+ * Categories: text, about, media
+ * Keywords: about, text
+ * Description: A section with two images, offset text and link.
+ *
+
+?>
+<!-- wp:group {"metadata":{"categories":["intro"],"name":"Two image row, offset text and link","patternName":"a8c/intro-two-column-with-image-and-content"},"align":"full","className":"alignfull is-style-default","style":{"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+<div class="wp-block-group alignfull is-style-default" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:spacer {"height":"var:preset|spacing|40"} -->
+<div style="height:var(--wp--preset--spacing--40)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"isStackedOnMobile":false,"align":"wide"} -->
+<div class="wp-block-columns alignwide is-not-stacked-on-mobile"><!-- wp:column {"width":"70%"} -->
+<div class="wp-block-column" style="flex-basis:70%"><!-- wp:cover {"url":"https://kiri.mystagingwebsite.com/wp-content/uploads/2025/02/pattern-vintage-bracelet.jpg","id":483,"alt":"A vintage silver cuff bracelet with square turquoise stone inlays.","dimRatio":0,"isDark":false,"style":{"color":[]}} -->
```

Comment:
This `src` attribute needs to be updated to point to the themes assets directory (`<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/pattern-vintage-bracelet.jpg`) before merging.